### ISSUE TITLE
can we support "auto  alphabetical order under" when add member in local ?

### ISSUE THREAD

**Issue by SAMZONG** [istio/community#1092]
like this.

- first, add your name in org/memeber.yaml
- second, run `make format org/memeber.yaml`

A simple script is all that's needed.

**Comment by howardjohn** [comment]
PRs welcome if there is a good way to do this. fwiw I normally use IDE function to do it, but simple script would be nice

**Comment by ronitblenz** [comment]
I would like to try this. Assign me :)

**Comment by ronitblenz** [comment]
Some updates:

The script is running fine!
```
go run sort_names.go
Names sorted successfully!
```
But the catch is that we cant have two main func in a directory. So, I had to make a subfolder for members to fix this issue. 

Now I have to make changes in the ```config_test.go```.

**Comment by ronitblenz** [comment]
Hey @howardjohn @SAMZONG 

I discovered a Quick Fix for it. I have made a directory called members, where you can add your name randomly. Now the script sorts the YAML and makes a copy of the ```tempname.yaml``` to generate updated ```members.yaml```.

<img width="1440" alt="Screenshot 2023-06-30 at 1 27 53 PM" src="https://github.com/istio/community/assets/91361382/6e901349-cfe1-4f9e-ab51-ce854f212ba7">

**Comment by ericvn** [comment]
I don't think we want to have people enter their names in multiple places. This leads to potential issues with changes overriding others or changes being lost. Maybe you have a script to grab the section from the member.yaml, sort it and put it back.

And how are you adding the updated list back into the PR. Are you having the automation create a new commit with the updated list?
